_includes/header.html

```diff
@@ -1,36 +1,30 @@
 <header>
+{% if page.theme %}<section class="{{ page.theme }}">{% endif %}
```

Comment:
For every page, the section includes the navbar and the page content. it might make sense for `header.html` to only include the `#splash-nav` div and to move the section tags part to `_layouts/main.html`. So the layout would be something like:
```
 {% include head.html %}
 <section  {% if page.theme %} class="{{ page.theme }}" {% endif %} >
  	{% include header.html %}
  	{{content}}
 </section>
 {% include footer.html %}
```
Also, I think sometimes the sections have id's. For example, the code page section has `code` as its id, so that would also be good to take into account.

Reply:
yeah this makes much more sense thank you!